.travis.yml

```diff
@@ -1,6 +1,7 @@
 language: node_js
 node_js:
-  - "0.10"
```

Comment:
Node sass still officiallly supports 0.10, 0.12, and iojs. It's not tested in Travis but is tested locally before releases.